src/pages/BookList.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import AddBooks from '../components/books/AddBooks';
+import Books from '../components/books/Books';
+
+const BookList = () => {
```

Comment:
The user is unable to see a book list and so the Remove button to remove a book is not shown. Please make sure a user is able to see the book list and can remove a book from the list.